AppleShopDZ/AppleShopDZ/ProductViewController.swift

```diff
@@ -187,10 +188,23 @@ final class ProductViewController: UIViewController {
         return label
     }()
     
+    private lazy var pdfImageView: UIImageView = {
+        let imageView = UIImageView(image: UIImage(systemName: Constants.documentImage))
+        imageView.isUserInteractionEnabled = true
+        imageView.frame = CGRect(x: view.bounds.width - 140, y: 147, width: 15, height: 15)
+        imageView.tintColor = .systemGray2
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+    
     // MARK: - Public property
     var productInfo: Product?
     
     // MARK: - LifeCycle
+    override func viewWillAppear(_ animated: Bool) {
+        tabSettings()
```

Comment:
```suggestion
        super. viewWillAppear(animated)
        tabSettings()
```
viewDidLoad
viewWillAppear